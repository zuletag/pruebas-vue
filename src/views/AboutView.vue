<template>
  <div>
    <h1>Nosotros</h1>
    <p>Bienvenidos</p>
  </div>
</template>

<script>
export default {
  name: 'AboutView',
};
</script>
