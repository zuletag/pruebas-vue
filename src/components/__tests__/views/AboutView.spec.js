import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'
import AboutView from '@/views/AboutView.vue'

describe('AboutView.vue', () => {
  it('Carga correctamente', () => {
    const wrapper = mount(AboutView)
    expect(wrapper.text()).toContain('Nosotros')
    expect(wrapper.text()).toContain('Bienvenidos')
  })
})

describe('AboutView.vue', () => {
  it('matches snapshot', () => {
    const wrapper = mount(AboutView)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
