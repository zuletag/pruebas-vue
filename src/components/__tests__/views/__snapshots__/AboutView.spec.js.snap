// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AboutView.vue > matches snapshot 1`] = `
"<div>
  <h1>Nosotros</h1>
  <p>Bienvenidos</p>
</div>"
`;
